server/src/main/java/org/opensearch/index/cache/request/RequestCacheStats.java

```diff
@@ -48,69 +52,127 @@
  */
 public class RequestCacheStats implements Writeable, ToXContentFragment {
 
-    private long memorySize;
-    private long evictions;
-    private long hitCount;
-    private long missCount;
+    private Map<String, StatsHolder> map;
```

Comment:
Maybe initialize this map inline here. This way you don't need to worry about this not being intialized.
Like
```
map = new HashMap<>(){{
       for(TierType tierType: TierType.values()) {
           put(tierType.getStringValue(), new StatsHolder());
       }
    }};
```